test/test_jit.py

```diff
@@ -8065,6 +8065,18 @@ def test_rand():
             return a + 1.0 - a
 
         self.checkScript(test_rand, ())
+        fn = torch.jit.script(test_rand)
```

Comment:
Can you add a comment for these tests explaining what they are intended to test?